src/main/views/manage-third-party-users-subscriptions.njk

```diff
@@ -0,0 +1,68 @@
+{% from "./macros/common-components.njk" import goBack, submitButton %}
+{% from "govuk/components/table/macro.njk" import govukTable %}
+{% from "govuk/components/button/macro.njk" import govukButton %}
+{% from "govuk/components/radios/macro.njk" import govukRadios %}
+{% from "govuk/components/error-summary/macro.njk" import govukErrorSummary %}
+
+{% extends "template.njk" %}
+{% block pageTitle %}
+  {{ title }}
+{% endblock %}
+{% block beforeContent %}
+  {{ super() }}
+  {{ goBack() }}
+{% endblock %}
+{% block content %}
+  <h1 class="govuk-heading-l">{{ heading }}</h1>
+  <form method="post">
+    <input type="hidden" id="userId" name="userId" value="{{ userId }}">
+    {{ govukRadios({
+      name: "channel",
+      fieldset: {
+        legend: {
+          text: channelText,
+          isPageHeading: true,
+          classes: "govuk-fieldset__legend--m"
+        }
+      },
+      items: channelItems,
+      errorMessage: null
```

Comment:
dont need this line here if always no error message?

Reply:
Good spot, I've removed. I was going to do an error message on this page but realised it could never happen, and forgot to remove this :P